backend/src/main/java/com/htecgroup/skynest/annotation/CanBucketBeEdited.java

```diff
@@ -0,0 +1,22 @@
+package com.htecgroup.skynest.annotation;
+
+import com.htecgroup.skynest.annotation.validator.EditBucketValidator;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.PARAMETER})
+@Constraint(validatedBy = EditBucketValidator.class)
+public @interface CanBucketBeEdited {
```

Comment:
UserCanEditBucket? CurrentUserCanEditBucket?

Reply:
any suggestions?

Reply:
Changed to CurrentUserCanEditBucket, gives most info.